src/app/theme.config.ts

```diff
@@ -1,27 +1,48 @@
 export const THEMES = {
-
   default: {
-    primaryColor: "#060461",
-    secondaryColor: "#4A227C",
-    bodyBackground: "#F6F8FC",
-    cardBackground: "#FFFFFF",
-    tagsBackground: "#F4ECFF",
-    navLabelColor: "#B275FF",
-    headerColor: "#4A227C",
-    headerLinkColor: "#F4ECFF",
-    primaryTextColor: "#000000",
-    secondaryTextColor: "#8D9091",
+    primaryColor: '#060461',
+    secondaryColor: '#4A227C',
+    cardBackground: '#FFFFFF',
+    tagsBackground: '#F4ECFF',
+    navLabelColor: '#B275FF',
+    headerColor: '#4A227C',
+    headerLinkColor: '#F4ECFF',
+    primaryTextColor: '#494949',
+    secondaryTextColor: '#8D9091',
+    btnNameColor: '#5526f5',
+    headerPrimaryColor: '#1b1818',
+    logoHeadlineColor: '#79787c',
+    logoSecondaryTextColor: '#ee8908',
+    menuiconcolor: '#5526f5',
+    btnPrimaryColor: '#060461',
+    headingColor: '#060461',
+    subheadingColor: '#212529',
+    searchBoxBorder: '#efefef',
+    hamBackground: '#f6f8fc',
+    bodyBackgroundd: '#a4a8ff3d',
+    bodysecondaryBackground: '#ffa4e042',
```

Comment:
bodysecondaryBackground to bgSecondaryColor

Reply:
resolved all